src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/TypeEditor.tsx

```diff
@@ -66,15 +66,16 @@ const TypeEditor = ({
     // is it not already terminated with an index?
     if (
       !getIndexFromPath(label) &&
-      !values?.length &&
-      isComponentDataType(type)
+      // !values &&
```

Comment:
Should we clear out these comments or did you need them for something in the next story?

Reply:
clearing it